bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -511,20 +505,41 @@
 
 	<channel-type id="airquality">
 		<item-type>String</item-type>
-		<label>Air quality level</label>
+		<label>Air Quality</label>
 		<description>Current air quality level based on volatile organic compounds (VOCs) measurement. Example: good or poor,
 			...</description>
-		<state readOnly="true" pattern="%s"></state>
+		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="airqualityppb">
 		<item-type>Number:Dimensionless</item-type>
-		<label>Air quality in ppb</label>
+		<label>Air Quality (ppb)</label>
 		<description>Current air quality based on measurements of volatile organic compounds (VOCs). The measured value is
 			specified in ppb (parts per billion).</description>
-		<state readOnly="true" pattern="%d"></state>
+		<state readOnly="true"/>
 	</channel-type>
 
+	<thing-type id="moisturesensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="deconz"/>
+		</supported-bridge-type-refs>
+		<label>Moisture Sensor</label>
+		<channels>
+			<channel typeId="moisture" id="moisture"/>
+			<channel typeId="last_updated" id="last_updated"/>
+		</channels>
+
+		<representation-property>uid</representation-property>
+
+		<config-description-ref uri="thing-type:deconz:sensor"/>
```

Comment:
Can we add this new thing to the [textual configuration example](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.deconz/README.md#things-file)? This would particularly useful since it has channel configuration for which the syntax might be unfamiliar or hard to remember.

Reply:
"Someone" can sure do it. Whoever wants to use figure out how to do it can do so, but I won't waste my time to make it easier for people who deliberately chose the hard way. Use discovery and managed things and you'll have no issues.